cloudflare/provider.go

```diff
@@ -164,6 +164,8 @@ func Provider() terraform.ResourceProvider {
 			"cloudflare_zone_dnssec":                            resourceCloudflareZoneDNSSEC(),
 			"cloudflare_notification_policy":                    resourceCloudflareNotificationPolicy(),
 			"cloudflare_notification_policy_webhooks":           resourceCloudflareNotificationPolicyWebhooks(),
+			"cloudflare_split_tunnel_include":									 resourceCloudflareSplitTunnelInclude(),
+			"cloudflare_split_tunnel_exclude":									 resourceCloudflareSplitTunnelExclude(),
```

Comment:
what benefits do we get from splitting (pun only partially intended 😛 ) these two? would it make more sense to have this as an attribute on a single, shared resource? example:

```hcl
resource "cloudflare_split_tunnel" "..."
  account_id = "..."
  type = "include"
  tunnels {
    host = "example.com"
    description = "..."
  }
}
```

i'm thinking we can combine these into a single resource and just use the parameter to swap the calling method. what do you think?

Reply:
I was mostly trying to future-proof it in case `include` and `exclude` ever behaved differently. Right now I can't imagine why that would be the case, but that's also often how these things go.